packages/lab/src/system-status/SystemStatus.tsx

```diff
@@ -0,0 +1,50 @@
+import { forwardRef, HTMLAttributes } from "react";
+import { makePrefixer } from "@salt-ds/core/src/utils";
+import {
+  StatusIndicator,
+  ValidationStatus,
+} from "@salt-ds/core/src/status-indicator";
+import { clsx } from "clsx";
+
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+
+import systemStatusCss from "./SystemStatus.css";
+
+export interface SystemStatusProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * Emphasize the styling by applying a background color: defaults to false
+   */
+  /**
+   *  A string to determine the current state of the SystemStatus
+   */
+  status?: ValidationStatus;
+}
+
+const withBaseName = makePrefixer("saltSystemStatus");
+
+export const SystemStatus = forwardRef<HTMLDivElement, SystemStatusProps>(
+  function SystemStatus(
+    { children, className, status = "info", ...rest },
```

Comment:
We're transitioning other components into possible "status-less". Should this have a variant without status? (gray)?
cc @shey-v

Reply:
no, think for now, it should always represent a status which is somewhat high-affordance.. so can't see and haven't seen a use-case for a more neutral variant.